src/pkg/keystore/keystore.go

```diff
@@ -12,31 +14,43 @@ import (
 
 type KeyStore struct {
 	storage storage.Storager
+	cfg     *config.Config
```

Comment:
There shouldn't be the global config because the keystore shouldn't know anything about other parameters except its parameters.